### DISCUSSION TITLE
Loco Positioning Deck with CrazySwarm

### DISCUSSION THREAD

**Discussion by semihsahhinx** [USC-ACTLab/crazyswarm#423] [Q&A]
Hello,
First of all, thanks for your great work. 
I have more than one CFs with Loco Positioning Deck. In CrazySwarm documentation, there is no spesific information about Loco Positioning Deck. So, how can I get these CFs positions with Loco Positioning Deck? Thanks in advance.

**Comment by whoenig** [comment]
You need to select the `motion_capture_type" to be "None", see https://crazyswarm.readthedocs.io/en/latest/configuration.html#select-hardware-make.

**Reply by semihsahhinx** [reply]
Well, I have another question. How can I access these positioning informations? I mean, how can I demonstrate it?

**Reply by whoenig** [reply]
In your launch file, set `enable_logging` and `enable_logging_pose` both to True. Then you can visualize the current state estimate in rviz.
